src/components/patient/patients/patientContainer.tsx

```diff
@@ -20,6 +21,7 @@ const PatientContainer: React.FC<PatientContainerProps> = props => {
   const [filteredPatients, setFilteredPatients] = useState<Patient[]>([]);
   const [isFetching, setIsFetching] = useState(true);
   const [searchQuery, setSearchQuery] = useState('');
+  const [patientsInDebtOrCredit, setPatientsInDebtOrCredit] = useState(Dictionary.patientContainer.showAllPatients);
```

Comment:
you should use an enum to in state, not the value from dictionary, I can show you